src/ssl.c

```diff
@@ -36304,12 +36327,28 @@ const char* wolfSSL_EC_curve_nid2nist(int nid)
     const WOLF_EC_NIST_NAME* nist_name;
     for (nist_name = kNistCurves; nist_name->name != NULL; nist_name++) {
         if (nist_name->nid == nid) {
-            return kNistCurves->name;
+            return nist_name->name;
         }
     }
     return NULL;
 }
 
+/**
+ * return nist curve id
+ * @param name nist curve name
+ * @return nist curve id when find, 0 when not find
```

Comment:
"when find" should be "when found".  "when not find" should be "when not found".

Reply:
Thanks!